<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="0" skipped="4">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-20T08:52:14 IST" name="Suite" finished-at="2021-08-20T08:52:14 IST" duration-ms="597">
    <groups>
      <group name="Opportunity">
        <method signature="CreateNewOpportunity.CreateNewOpportunityTest(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.CreateNewOpportunity@5afa3c9]" name="CreateNewOpportunityTest" class="testNg.CreateNewOpportunity"/>
        <method signature="CreateNewOpportunityWithoutMandatoryFields.CreateNewOpportunityWithoutMandatoryFieldsTest(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.CreateNewOpportunityWithoutMandatoryFields@710c2b53]" name="CreateNewOpportunityWithoutMandatoryFieldsTest" class="testNg.CreateNewOpportunityWithoutMandatoryFields"/>
        <method signature="EditOpportunity.EditOpportunityTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.EditOpportunity@6fa51cd4]" name="EditOpportunityTest" class="testNg.EditOpportunity"/>
        <method signature="DeleteOpportunity.DeleteOpportunityTest(java.lang.String,java.lang.String)[pri:0, instance:testNg.DeleteOpportunity@2767e23c]" name="DeleteOpportunityTest" class="testNg.DeleteOpportunity"/>
      </group> <!-- Opportunity -->
    </groups>
    <test started-at="2021-08-20T08:52:14 IST" name="Test" finished-at="2021-08-20T08:52:14 IST" duration-ms="597">
      <class name="testNg.CreateNewOpportunity">
        <test-method signature="CreateNewOpportunityTest(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.CreateNewOpportunity@5afa3c9]" started-at="2021-08-20T08:52:14 IST" name="CreateNewOpportunityTest" data-provider="fecthData" finished-at="2021-08-20T08:52:14 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)
at org.testng.internal.Parameters.handleParameters(Parameters.java:798)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException
at org.apache.poi.xssf.usermodel.XSSFWorkbook.getSheet(XSSFWorkbook.java:1120)
at testNg.ReadExcel.excelData(ReadExcel.java:13)
at testNg.baseClass.getData(baseClass.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)
... 12 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateNewOpportunityTest -->
      </class> <!-- testNg.CreateNewOpportunity -->
      <class name="testNg.EditOpportunity">
        <test-method signature="EditOpportunityTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.EditOpportunity@6fa51cd4]" started-at="2021-08-20T08:52:14 IST" name="EditOpportunityTest" finished-at="2021-08-20T08:52:14 IST" depends-on-methods="testNg.CreateNewOpportunity.CreateNewOpportunityTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EditOpportunity.EditOpportunityTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.EditOpportunity@6fa51cd4] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EditOpportunity.EditOpportunityTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.EditOpportunity@6fa51cd4] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EditOpportunityTest -->
      </class> <!-- testNg.EditOpportunity -->
      <class name="testNg.DeleteOpportunity">
        <test-method signature="DeleteOpportunityTest(java.lang.String,java.lang.String)[pri:0, instance:testNg.DeleteOpportunity@2767e23c]" started-at="2021-08-20T08:52:14 IST" name="DeleteOpportunityTest" finished-at="2021-08-20T08:52:14 IST" depends-on-methods="testNg.EditOpportunity.EditOpportunityTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DeleteOpportunity.DeleteOpportunityTest(java.lang.String,java.lang.String)[pri:0, instance:testNg.DeleteOpportunity@2767e23c] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DeleteOpportunity.DeleteOpportunityTest(java.lang.String,java.lang.String)[pri:0, instance:testNg.DeleteOpportunity@2767e23c] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeleteOpportunityTest -->
      </class> <!-- testNg.DeleteOpportunity -->
      <class name="testNg.CreateNewOpportunityWithoutMandatoryFields">
        <test-method signature="CreateNewOpportunityWithoutMandatoryFieldsTest(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testNg.CreateNewOpportunityWithoutMandatoryFields@710c2b53]" started-at="2021-08-20T08:52:14 IST" name="CreateNewOpportunityWithoutMandatoryFieldsTest" data-provider="fecthData" finished-at="2021-08-20T08:52:14 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)
at org.testng.internal.Parameters.handleParameters(Parameters.java:798)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException
at org.apache.poi.xssf.usermodel.XSSFWorkbook.getSheet(XSSFWorkbook.java:1120)
at testNg.ReadExcel.excelData(ReadExcel.java:13)
at testNg.baseClass.getData(baseClass.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)
... 12 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateNewOpportunityWithoutMandatoryFieldsTest -->
      </class> <!-- testNg.CreateNewOpportunityWithoutMandatoryFields -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
